examples/sst2/input_pipeline.py

```diff
@@ -0,0 +1,271 @@
+# Copyright 2020 The Flax Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""SST-2 input pipeline."""
+
+import collections
+from typing import Dict, List, Set, Text, Tuple
+import os
+
+from absl import logging
+
+import numpy as np
+import tensorflow.compat.v2 as tf
+import tensorflow_datasets as tfds
+
+
+# pylint: disable=no-member
```

Comment:
Why do we need this?

Reply:
TF has a strange way of defining inheritance and pylint doesn't get it. (Some linters don't check this deep though.)